R/create_wpa_naive.R

```diff
@@ -87,9 +89,25 @@ create_wpa_naive <- function(df, wp_model = cfbscrapR:::wp_model) {
   df = df %>% 
     dplyr::arrange(.data$game_id, .data$new_id)
   
+  
+  
   Off_Win_Prob = as.vector(predict(wp_model, newdata = df, type = "response"))
   df$wp_before = Off_Win_Prob
-
+  # Kickoff plays
+  # Calculate EP before at kickoff as what happens if it was a touchback
```

Comment:
This is for WP not EP right? Can we update comment. I know it was a copy/paste thing

Reply:
So WP on kickoffs is compared relative to a touchback?

Reply:
Yep, just a copy paste non-change